chalicelib/checks/wrangler_checks.py

```diff
@@ -541,15 +541,15 @@ def change_in_item_counts(connection, **kwargs):
     # add deleted/replaced items
     search_query += '&status=deleted&status=replaced'
     search_resp.extend(ff_utils.search_metadata(search_query, key=connection.ff_keys))
-    search_output = []
     for res in search_resp:
-        # convert type to index name. e.g. ExperimentSet --> experiment_set
-        res_index = convert_camel_to_snake(res['@type'][0])
-        if res_index in diff_counts:
-            diff_counts[res_index]['ES'] = diff_counts[res_index]['ES'] + 1 if 'ES' in diff_counts[res_index] else 1
+
+        # Stick with given type name in CamelCase since we need it to match ES type-name
+        _type = res['@type'][0]
+        if _type in diff_counts:
+            diff_counts[_type]['ES'] = diff_counts[_type]['ES'] + 1 if 'ES' in diff_counts[_type] else 1
         else:
             # db entry wasn't already present for this index
-            diff_counts[res_index] = {'DB': 0, 'ES': 1}
+            diff_counts[_type] = {'DB': 0, 'ES': 1}
```

Comment:
Is this the same as:
```
diff_counts[_type]['ES'] = diff_counts[_type].get('ES', 0) + 1
```
Seems like you could almosthave set up `diff_counts` to be a `defaultdict(lambda: {'DB': 0, 'ES': 0}` and then you would not need to have checked. But it depends on whether you want entries with all zeros to show up if nothing gets modified. Since it seems like you do, then perhaps better something like this (untested), which seems to be all that's going on here and is maybe easier to read, assuming I didn't make a mistake:
```
_type = res['@type'][0]
_entry = diff_counts.get(_type)
if not _entry:
    diff_counts[_type] = _entry = {'DB': 0, 'ES': 0}
if _type in diff_counts:
    _entry['ES'] += 1
```

I don't see unit tests in this PR so don't know if this is tested, but given that the shape of things is changing, I'd be fairly surprised if there were unit tests that spanned. I'd expect an existing unit test to be broken and need fixing if there were good ones.

Reply:
In case that was opaque, by "if nothing gets modified" I mean "if you never end up bumping the ES count above 0".

Reply:
There are no unit tests because these are foursight checks, they are only 'tested' when run manually from the terminal or in production (unfortunately). I like your change better though and will take it.